UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.cs

```diff
@@ -112,6 +112,12 @@ IEnumerator LoadSubScene(string sceneName, SubsceneLoadTimer loadTimer = null, b
 				RequestObserverRefresh.Send(sceneName);
 			}
 		}
+
+		loadedScenesList.Add(new SceneInfo
```

Comment:
Make sure to set is dirty,  if it's a Synch  list

Reply:
it already does in the function thats calling it... which good thing you reminded me; this can be further cleaned by moving that over to the load function

Reply:
I think the load function gets called on client?

Reply:
?